src/IO/H5/VolumeData.cpp

```diff
@@ -163,7 +166,22 @@ DataVector VolumeData::get_tensor_component(
       volume_file_root_group_.id(),
       "ObservationId" + std::to_string(observation_id) + "/" + grid_name,
       AccessType::ReadOnly);
-  return h5::read_data(spatial_group.id(), tensor_component);
+  const hid_t dataset_id =
+      h5::open_dataset(spatial_group.id(), tensor_component);
+  const hid_t dataspace_id = h5::open_dataspace(dataset_id);
+  auto Rank = static_cast<size_t>(H5Sget_simple_extent_ndims(dataspace_id));
+  h5::close_dataspace(dataspace_id);
+  h5::close_dataset(dataset_id);
+  switch (Rank) {
```

Comment:
Formatting of the switch case is weird.

Reply:
Explain...

Reply:
This is how I saw it done in examples online

Reply:
From our code review guidelines: "Adhere to Google style. Can use clang-format -style=google."

Reply:
I ran -style=google as a clang-format flag for these lines but it does not change the formatting

Reply:
```
  switch (Rank) {
    case 1:
      return h5::read_data<1, DataVector>(spatial_group.id(), tensor_component);
    case 2:
      return h5::read_data<2, DataVector>(spatial_group.id(), tensor_component);
    case 3:
      return h5::read_data<3, DataVector>(spatial_group.id(), tensor_component);
    default:
      ERROR("Rank must be 1, 2, or 3. Received data with Rank = " << Rank);
  }
```

This was formatted against the `.clang_format` file in the repo.